go.mod

```diff
@@ -26,3 +26,5 @@ require (
 	sigs.k8s.io/kustomize/cmd/config v0.8.5
 	sigs.k8s.io/kustomize/kyaml v0.9.4
 )
+
+replace sigs.k8s.io/cli-utils => ../../sigs.k8s.io/cli-utils
```

Comment:
After cli-utils is submitted, this will have to be updated and removed.

Reply:
Yep, I'll remove this after the PR kubernetes-sigs/cli-utils#270 is merged.

Reply:
kubernetes-sigs/cli-utils#270 is merged. I published a new version for cli-utils: v0.22.0 and used it in `go.mod`.